src/storage.js

```diff
@@ -0,0 +1,34 @@
+//get config
+require('./config');
+//get redis constant searches redis globally 
+
+const Redis = require('ioredis'); //chnage to ioredis
+//connecting client
+const client = new Redis(process.env.REDIS_URL);
+
+//result from require, displays externally
+module.exports = {
+    addFeed: function(name, url, f) {
+      const feed = { 'name': name, 'url': url }; //insted of json store id to a hash
+      client.pipeline().incr('user_id').get('user_id', function(err, id) {
```

Comment:
Does `incr('user_id')` work if it's never been set?  Also, should this be named `feed_id`?  It's not a user.